install.sh

```diff
@@ -1,42 +1,40 @@
-#!/bin/bash
+#!/bin/sh
 
-function main() {
-    if [[ -L "$HOME/.emacs" ]]; then
+chemacs_path=$(realpath "$(dirname "$0")")
+
+main() {
+    if [ -L "$HOME/.emacs" ]; then
         check_existing_symlink
     else
         create_symlink_if_possible
     fi
 }
 
-function check_existing_symlink() {
-    local target=$(readlink -f $HOME/.emacs)
+check_existing_symlink() {
+    target=$(readlink -f "$HOME/.emacs")
```

Comment:
nice catch!

Reply:
Well, thanks, but most corrections were pointed to me by shellcheck actually! :smile: